tests/Functional/Execution/VariablesTest.php

```
@@ -0,0 +1,248 @@
+<?php
+
+namespace Digia\GraphQL\Test\Functional\Execution;
+
+
+use Digia\GraphQL\Test\TestCase;
+use function Digia\GraphQL\execute;
+use function Digia\GraphQL\parse;
+use function Digia\GraphQL\Type\GraphQLString;
+use function Digia\GraphQL\Type\newGraphQLInputObjectType;
+use function Digia\GraphQL\Type\newGraphQLList;
+use function Digia\GraphQL\Type\newGraphQLNonNull;
+use function Digia\GraphQL\Type\newGraphQLObjectType;
+use function Digia\GraphQL\Type\newGraphQLScalarType;
+use function Digia\GraphQL\Type\newGraphQLSchema;
+
+
+/**
+ * Class VariablesTest
+ * @package Digia\GraphQL\Test\Functional\Execution
+ */
+class VariablesTest extends TestCase
+{
+    private $schema;
+
+
+    /**
+     * @throws \Digia\GraphQL\Error\InvalidTypeException
```

It’s better to ignore the errors than declare `@throws`.